Files/Dialogs/SettingsDialog.xaml.cs

```diff
@@ -29,26 +29,13 @@ private void Current_SizeChanged(object sender, WindowSizeChangedEventArgs e)
 
         private void UpdateDialogLayout()
         {
-            if (Window.Current.Bounds.Width <= 700)
-            {
-                SettingsPane.PaneDisplayMode = Microsoft.UI.Xaml.Controls.NavigationViewPaneDisplayMode.LeftCompact;
-                SettingsContentFrame.Width = 410;
-                Column0.Width = new GridLength(60);
-            }
-            else
-            {
-                SettingsPane.PaneDisplayMode = Microsoft.UI.Xaml.Controls.NavigationViewPaneDisplayMode.Left;
-                SettingsContentFrame.Width = 460;
-                Column0.Width = new GridLength(0, GridUnitType.Auto);
-            }
-
             if (Window.Current.Bounds.Height <= 600)
             {
-                ContainerGrid.Height = Window.Current.Bounds.Height;
+                SettingsScrollViewer.Height = Window.Current.Bounds.Height - 40;
```

Comment:
Curious: why is it necessary to manually set the ScrollViewer height?

Reply:
It's the only way I was able to get it to stretch.